controller/readonly.go

```diff
@@ -96,14 +95,14 @@ func (ctrl *ReadOnly) checkInspect(ctx context.Context, d driver.Driver) (bool,
 	// rendering a template may take several cycles in order to completely fetch
 	// new data
 	if rendered {
-		log.Printf("[TRACE] (ctrl) template for task %q rendered: %+v", taskName, rendered)
+		ctrl.logger.Trace("template for task rendered", taskNameLogKey, taskName)
 
-		log.Printf("[INFO] (ctrl) inspecting task %s", taskName)
+		ctrl.logger.Info("inspecting task", taskName)
```

Comment:
I think it's missing the key?
```suggestion
		ctrl.logger.Info("inspecting task", taskNameLogKey, taskName)
```